README.md

```diff
@@ -3,24 +3,38 @@ An LC-3 virtual machine written in Rust, based on [Write your Own Virtual Machin
 
 ## Usage
 
-1. Clone the repository.
+1. `just build` to build and install the binary, or manually use `cargo`.
 
 2. Run:
 
+    - For help:
+
+    ```sh
+    lc3-vm -h
+    ```
+
+    - With an object file:
+
+    ```sh
+    lc3-vm -i <path-to-obj>
+    ```
+
+## Justfile
```

Comment:
Maybe you could add a little hint to install `Just` in the system with `Brew`

Reply:
Right!